api/routes/system.py

```
@@ -0,0 +1,1224 @@
+        
+    except Exception as e:
+        # Log failed ping attempt
+        try:
+            db = PostgreSQLManager()
+            db.log_system_event(
+                event_type="worker_ping_failed",
+                description=f"Worker ping failed: {str(e)}",
+                user_id="admin",
+                metadata={"error": str(e)}
+            )
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)